src/main/java/io/vertx/codegen/generators/cheatsheet/DataObjectCheatsheetGen.java

```diff
@@ -119,8 +119,8 @@ private String getDataType(TypeInfo type) {
           return "Json object";
         case JSON_ARRAY:
           return "Json array";
-        case DATA_OBJECT:
-          return "link:dataobjects.html#" + type.getRaw().getSimpleName() + '[' + type.getRaw().getSimpleName() + ']';
+//        case DATA_OBJECT:
```

Comment:
Can we remove it?

Reply:
BTW now removing this one the cheatsheet is not generated anymore correctly, because the case `OTHER` handles only `Instant`

Reply:
will do, but I'm not sure we are going to keep cheatsheet in 4.0 doc

Reply:
If we don't keep it, then we can just remove it

Reply:
In another pr

Reply:
we are not sure yet as we have not discussed 4.0 docs more extensively